linetools/spectralline.py

```diff
@@ -425,8 +425,8 @@ def measure_restew(self, **kwargs):
         self.measure_ew(**kwargs)
 
         # Push to rest-frame
-        self.attrib['EW'] = self.attrib['EW'] / (self.attrib['z']+1)
-        self.attrib['sig_EW'] = self.attrib['sig_EW'] / (self.attrib['z']+1)
+        self.attrib['EW'] = self.attrib['EW'] / (self.z+1)
+        self.attrib['sig_EW'] = self.attrib['sig_EW'] / (self.z+1)
 
```

Comment:
Now that I think about it, shouldn't we have 'observed EW' and 'rest-frame EW' as separate attributes?